_posts/2017-10-06-rust-blog-series-2.md

```diff
@@ -0,0 +1,1144 @@
+---
+layout: post
+title:  "Making a simple blog with Rust: Part II"
+date:   2017-10-06
+categories:
+- rust
+- programming
+- rocket
+- diesel
+---
+
+Welcome to Part II in our Rust blog hacking adventure.
+In this post, we'll be learning about connecting to the database
+and using some very simple Diesel APIs to seed that database.
+At the end of this post, we should have completed two fundamental tasks
+1. Seed database (fill with initial dummy data) with users and posts,
+2. Output each post title and author to our index page.
+
+If you haven't checked out [Part I],
+please do as it covers project setup and will be referenced heavily throughout this guide.
+
+[Part I]: ./rust-blog-series-1.html
+
+## infer_schema!
+
+Our "entry point" for Diesel applications.
+The Diesel [schema in depth guide] reviews all the options for database schema in greater details.
+All we need to know for the purpose of the guide is...
+
+> infer_schema! is a macro provided by diesel_codegen when you have enabled the 
+> feature for one or more database backends.
+> The macro will establish a database connection at compile time,
+> query for a list of all the tables,
+> and generate infer_table_from_schema! for each one.
+> infer_schema will skip any table names which start with __.
+
+Using `infer_schema!` will give us some implicit benefits for joining tables later in development,
+but it's also very easy to get started with,
```

Comment:
Is the comma here needed ? It feels weird to me. (Feel free to disregard any comment related to english as you're a native speaker and I'm not :p)